tests/Microsoft.Identity.Test.ConfigurationProvider/Microsoft.Identity.Test.ConfigurationProvider.csproj

```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
I don't think any new project is required. We have too many test projects already.